### PR TITLE
Unit test 2.0

### PR DESCRIPTION
**Pull request by timkimadobe**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by cacheung** [comment]
Are you going to remove powermock for EdgeBridgeTest file?

File: code/edgebridge/build.gradle
Comment (emdobrin) [inline code/edgebridge/build.gradle:218]: can be removed, right?

File: code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java
Comment (emdobrin) [inline code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java:166]: we should keep the GMT timezone

Comment (emdobrin) [reply code/edgebridge/src/main/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtension.java:166]: also why not hiding the format and timezone under the TimeUtils API like we do in swift, ```e.g.: TimeUtils.getISO8601UTCDateWithMilliseconds(new Date(timeout))```

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (emdobrin) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:66]: - how are we testing the new method of registration?
- can we move these tests to EdgeBridgeTests since they test the testing the public APIs and not the internal EdgeBridgeExtension?

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (emdobrin) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:172]: this seems to be a generic note, do you intend to keep it?

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (emdobrin) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:129]: for all these asserts can we use the core eventype/source instead?

Comment (emdobrin) [reply code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:129]: I think for your tests you can assume core value will not change since they are publicly shared constants. In this case catching unexpectedly changed values should be part of core tests

File: code/edgebridge/build.gradle
Comment (emdobrin) [inline code/edgebridge/build.gradle:216]: do these libs evolve together, can we use rootProject.ext.mockitoVersion for both core and inline?

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (emdobrin) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:257]: can we also remove these values from EdgeBridgeTestConstants now?

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (emdobrin) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:762]: this should be GMT

File: code/build.gradle
Comment (emdobrin) [inline code/build.gradle:23]: nit: rename to mockitoVersion

**Review by emdobrin — COMMENTED** [review]
Thanks for the updates so far, changes look better. will be waiting for the pending comments before final review

File: code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java
Comment (cacheung) [inline code/edgebridge/src/test/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeExtensionTests.java:338]: Don't need to test null this case?

File: code/edgebridge/src/androidTest/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeTests.java
Comment (emdobrin) [reply code/edgebridge/src/androidTest/java/com/adobe/marketing/mobile/edge/bridge/EdgeBridgeTests.java]: thanks for the updates, what you have right now under the unit tests looks good to me

**Review by emdobrin — APPROVED** [review]

**Pull request merged**
